editor/editor_node.cpp

```diff
@@ -6417,8 +6417,10 @@ EditorNode::EditorNode() {
 
 	p->add_separator();
 	p->add_shortcut(ED_SHORTCUT("editor/export", TTR("Export...")), FILE_EXPORT_PROJECT);
+#ifndef ANDROID_ENABLED
 	p->add_item(TTR("Install Android Build Template..."), FILE_INSTALL_ANDROID_SOURCE);
 	p->add_item(TTR("Open User Data Folder"), RUN_USER_DATA_FOLDER);
```

Comment:
Isn't this possible/relevant on Android too? Or is that in a scope that the file manager apps aren't allowed to access?

Reply:
> Or is that in a scope that the file manager apps aren't allowed to access

Correct, the user data folder is in an app-private directory provided by [`Context.getFilesDir()`](https://developer.android.com/reference/android/content/Context#getFilesDir()) that's not accessible to file manager.

The alternative would be to move that directory to the one provided by [`Context.getExternalFilesDir(...)`](https://developer.android.com/reference/android/content/Context#getExternalFilesDir(java.lang.String)).
The tradeoff being that directory may not be available on low-end devices, or may be slower (e.g: removable sdcard).